.github/workflows/ci.yml

```diff
@@ -15,3 +15,7 @@ jobs:
   call-terraform-ci-pipeline:
     uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.8.1
     secrets: inherit
+    with:
```

Comment:
sorry about this but since this is new, we should use the SCCv2 which was just added. https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/pull/147/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR20
Let me know if you have any questions. These are the first modules to use it